docs/tutorial/population_dynamics.rst

```diff
@@ -124,55 +173,82 @@ bins to be plotted as `n_bins`.
+exponential model, but only slightly so it is not strong evidence to choose the more complex model.
 
-Going back to our experimental data, we can next attempt to estimate confidence intervals (CI) for the parameters using bootstrapping.
-Here, a random dataset with the same size as the original is sampled (with replacement) from the original dataset. This sampled dataset
+As an additional check, we can estimate confidence intervals (CI) for the parameters using bootstrapping.
+Here, a dataset with the same size as the original is randomly sampled (with replacement) from the original dataset. This random sample
 is then fit using the MLE method, just as for the original dataset. The fit results in a new estimate for the model parameters.
 This process is repeated many times, and the distribution of the resulting parameters can be analyzed to estimate certain statistics about the them::
 
-    dwell_2.calculate_bootstrap(iterations=1000)
+    dwell_2.calculate_bootstrap(iterations=100)
+
+    plt.figure()
     dwell_2.bootstrap.plot(alpha=0.05)
+    plt.show()
 
-.. image:: kymo_bind_bootstrap_2.png
+.. image:: figures/population_dynamics/dwell2_bootstrap.png
 
-Here we see the distributions of the bootstrapped parameters. The vertical lines indicate the
-means of the distributions, while the red area indicates the estimated confidence intervals. The `alpha` argument determines
-the CI that is estimated as `100*(1-alpha)` % CI; in this case we're showing the estimate for the 95% CI. The values for the
-lower and upper bounds are the `100*(alpha/2)` and `100*(1-alpha/2)` percentiles of the distributions.
+Here we see the distributions of the bootstrapped parameters, each of which ideally should look like a Normal (Gaussian) distribution.
+The vertical lines indicate the means of the distributions, while the red area indicates the estimated confidence intervals.
+The `alpha` argument determines the CI that is estimated as `100*(1-alpha)` % CI; in this case we're showing the estimate for the 95% CI.
+The values for the lower and upper bounds are the `100*(alpha/2)` and `100*(1-alpha/2)` percentiles of the distributions.
 
-Note, however, that while the means correspond well with the optimized model parameters, the distributions are not symmetric.
-In such a case, the simple method of using percentiles as CI values may not be appropriate. For more advanced analysis,
-the distribution values are directly available through the properties `DwelltimeModel.bootstrap.amplitude_distributions` and
-`DwelltimeModel.bootstrap.lifetime_distributions` which return the data as a `numpy` array with
-shape `[# components, # bootstrap samples]`.
+Clearly the distributions here are not Gaussian. Specifically, the two distributions on the left for the fractional amplitudes
+are split, indicating that the model parameters are non-identifiable (there are different combinations of parameters that give
```

Comment:
Aren't they always going to potentially be bimodal in this setup; even if the rates are identifiable? One can always exchange `a1` with `a2` and `k1` with `k2` at the same time. Or do you guarantee one is always bigger than the other? Would it make sense to sort them after optimization, because otherwise the ci's will also always be unnecessarily wide (since you're gonna see both modes for both parameters).

I don't think it's sufficient to conclude that it's non-identifiable. It could be multimodal, but those modes could be well defined. I would omit this statement.

For the complexity question, it seems far more telling that many of the alpha estimates go to (near) zero (e.g. one component (almost) disabled).

For whether the complexity is warranted, I would probably do a likelihood ratio test for this. The models are nested (you can turn the bigger model into the smaller one). Basically, what you would do with an LR test is set component 2 to zero, calculate the log-likelihood. Then take the best fit you got from your bootstrap and calculate the ratio of the two log-likelihoods. This ratio is asymptotically chi-squared distributed with the number of degrees of freedom set to the number of parameters difference between the two models.